tensorrt_llm/_torch/pyexecutor/py_executor.py

```diff
@@ -876,7 +876,7 @@ def _executor_loop(self):
                 self._pad_attention_dp_dummy_request()
 
                 if self.draft_model_engine is not None or self.drafter is not None:
-                    self._prepare_draft_requests()
+                    self._prepare_draft_requests(self.active_requests)
```

Comment:
`_prepare_draft_requests `is a class function and can access `self.active_requests`. No need to pass the parameter here.